kujira/chain.json

```diff
@@ -9,18 +9,122 @@
   "bech32_prefix": "kujira",
   "daemon_name": "kujirad",
   "node_home": "$HOME/.kujira",
-  "key_algos": [
-    "secp256k1"
-  ],
+  "key_algos": ["secp256k1"],
   "slip44": 118,
   "fees": {
     "fee_tokens": [
       {
         "denom": "ukuji",
-        "fixed_min_gas_price": 0.00119,
-        "low_gas_price": 0.01,
-        "average_gas_price": 0.025,
-        "high_gas_price": 0.03
+        "fixed_min_gas_price": 0.0034,
+        "low_gas_price": 0.0034,
+        "average_gas_price": 0.0051,
+        "high_gas_price": 0.00681
+      },
+      {
+        "denom": "factory/kujira1qk00h5atutpsv900x202pxx42npjr9thg58dnqpa72f2p7m2luase444a7/uusk",
+        "fixed_min_gas_price": 0.01186,
+        "low_gas_price": 0.01186,
+        "average_gas_price": 0.01779,
+        "high_gas_price": 0.02372
+      },
+      {
+        "denom": "ibc/295548A78785A1007F232DE286149A6FF512F180AF5657780FC89C009E2C348F",
```

Comment:
These denoms are not found. Please ensure all these assets are defined in kujira's assetlist.json